tests/sonic-sw-dp-bring-down.yaml

```diff
@@ -18,23 +18,23 @@
    - name: Stop SONiC VPP Container 
      shell: | 
        cd sonic-vpp
-       ./start_sonic_vpp.sh stop -n sonic-vpp
+       NO_LINUX_NL={{ no_linux_nl }} ./start_sonic_vpp.sh stop -n sonic-vpp
```

Comment:
stop command does not require passing the env variable NO_LINUX_NL. You can remove the env in all the stop instances.

Reply:
Removed.